cypress/features/regression/test/drawer.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Drawer component
+  I want to test Drawer component
+
+  Background: Open Drawer component page
+    Given I open basic Test "Drawer" component page
+
+  @positive
+  Scenario: Expanding Drawer
+    Given I click on Drawer arrow
+    Then sidebar should have class open
+      And toggle icon switched orientation to open
+      And sidebar text is visible
+
+  @positive
+  Scenario: Collapsing Drawer
+    Given I click on Drawer arrow
+      And I click on Drawer arrow
```

Comment:
try to create step like `I click on Drawer arrow {int} times` . Then you can remove `And` step and in the `Given` pass 2

Reply:
fixed 4d965d0